lib/crowdmap_averages.rb

```diff
@@ -0,0 +1,105 @@
+require_dependency 'aircasting/username_param'
```

Comment:
Do you know you can use `git mv oldfilename newfilename` to rename files. This way git can recognize the connection even when the renamed file was edited. Makes looking at PR and searching git history easier :)
https://help.github.com/en/articles/renaming-a-file-using-the-command-line

Reply:
Nice tip, thanks!